conf/root.go

```diff
@@ -101,5 +134,77 @@ var (
 func init() {
 	createContextCmd.Flags().StringVarP(&confFileTemplate, "templatefile", "f", "", "-f <config file to build context from>")
 	dumpCmd.Flags().StringVarP(&dumpOutfile, "outfile", "o", "", "-o <outfile")
-	RootCmd.AddCommand(dumpCmd, createContextCmd, getContextCmd, getContextsCmd, useContextCmd, deleteContextCmd)
+	RootCmd.AddCommand(dumpCmd, createContextCmd, currentContextCmd, getContextsCmd, useContextCmd, deleteContextCmd, setCmd, editCmd)
+}
+
+func setConfigWithViper(key string, value string) error {
```

Comment:
not all of our "value"s can be strings right? what about numeric types?

Reply:
viper.Unmarshal automatically accounts for the correct types and returns an error if there is a mismatch - I tested this by setting port to "8asdf" and it failed as expected.